index.js

```diff
@@ -0,0 +1,33 @@
+const defaults = require('lodash.defaults');
+const request = require('request');
+const defaultOptions = {
+  verbose: false,
+  // todo: should these be env variables?
+  host: 'localhost', // the micro-mail host
```

Comment:
nope, options passed in is fine.  we should get rid of port though and pass in the entire string: `https://micro-mail.com:8080`